csharp/src/SeedLang/Runtime/NativeFunction.cs

```diff
@@ -40,16 +43,77 @@ internal class NativeFunction : IFunction {
 
   // The static class to define all the build-in native functions.
   internal static class NativeFunctions {
+    public const string HasNext = "hasnext";
+    public const string Iter = "iter";
```

Comment:
Python:
```
>>> iter([1, 2])
<list_iterator object at 0x106cf2730>
```

SeedPython:
```
>>> iter ([1, 2])
---------- Source ----------
iter ([1, 2])

---------- Run ----------
iter ([1, 2])
iter ([1, 2])
: 20220209231748914 Fatal (SeedLang.VM) <> [Ln 1, Col 0 - Ln 1, Col 12] 19: RuntimeErrorInvalidCast
```

Reply:
`iter` build-in function is removed, because I realized that there is an easy way to optimize `for in` statements execution, and without the need of `iter`. The new method is several times faster than my previous implementation.